pkg/cmd/controller/config.go

```diff
@@ -35,9 +37,11 @@ const (
 	infraServiceAccountControllerServiceAccountName             = "serviceaccount-controller"
 	iInfraServiceAccountPullSecretsControllerServiceAccountName = "serviceaccount-pull-secrets-controller"
 	infraBuildControllerServiceAccountName                      = "build-controller"
+	infraBuilderRoleBindingsControllerServiceAccountName        = "builder-rolebindings-controller"
```

Comment:
I would omit this - these are service accounts created by some pretty ancient "bootstrap" logic in openshift-apiserver. (_I think_). Don't assume these exist.